cvat/apps/events/views.py

```diff
@@ -31,11 +34,15 @@ def create(self, request):
         serializer = ClientEventsSerializer(data=request.data, context={"request": request})
         serializer.is_valid(raise_exception=True)
 
-        for event in serializer.data["events"]:
-            message = JSONRenderer().render(event).decode('UTF-8')
+        handle_client_events_push(request, serializer.validated_data)
```

Comment:
I think this should be after the event loop, so that the working time events are logged after the events that were used to generate them.

Reply:
I do not think that is so important
Timestamp of generated events == timestamp of the first event in sequence used to generate